terraform/modules/aws-cargohold-asset-storage/variables.tf

```diff
@@ -0,0 +1,17 @@
+variable "region" {
```

Comment:
This is misleading. While the output outputs a static string (which is not really useful) containing this region, nothing in this module actually ensures that the bucket is stored in that region. So if the user of the module configures their provider to use another region, this will be wrong. 

A few options I see:

1. Remove this variable and output
2. Use the region for the bucket creation in "aws_s3_bucket". There is a `region =` argument.

In all cases, the output should use the region from the "aws_s3_bucket" attributes, not from the input variable (as that could change, and then be wrong). See https://www.terraform.io/docs/providers/aws/r/s3_bucket.html#attributes-reference

Reply:
thx very much for pointing this out!